Makefile

```diff
@@ -202,23 +202,17 @@ release: eksctl-build-image ## Create a new eksctl release
 	    $(EKSCTL_BUILDER_IMAGE) \
 	      ./do-release.sh
 
-JEKYLL := docker run --tty --rm \
-  --name=eksctl-jekyll \
-  --volume="$(CURDIR)":/usr/src/app \
-  --publish="4000:4000" \
-    starefossen/github-pages
-
 ##@ Site
+HUGO := $(GOBIN)/hugo
+HUGO_ARGS ?= --gc --minify
 
 .PHONY: serve-pages
 serve-pages: ## Serve the site locally
-	-docker rm -f eksctl-jekyll
-	$(JEKYLL) jekyll serve -d /_site --watch --force_polling -H 0.0.0.0 -P 4000
+	cd site/ ; $(HUGO) serve $(HUGO_ARGS)
 
 .PHONY: build-pages
-build-pages: ## Generate the site using jekyll
-	-docker rm -f eksctl-jekyll
-	$(JEKYLL) jekyll build --verbose
+build-pages: ## Generate the site
```

Comment:
I don't know how Hugo works. But can we have explicit file prerequisites (and maybe even an explicit file target(s)) for this?

Reply:
It's really fast (under 1s). I don't see a reason, and it would be too much effort - everything in `site` is a dependency, and it outputs lots of files.